exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,13 @@
 // Start by creating a variable `message`
+var myName = "Helen";
+var myNameLength = myName.length;
+var message = "Hello, my name is " + myName + "and my name is " + myNameLength + " characters long."
```

Comment:
This could be a good place to use string interpolation, i.e
```suggestion
var message = `Hello, my name is ${myName} and my name is ${myNameLength} characters long.`
```